tests/test_yield.py

```diff
@@ -0,0 +1,14 @@
+
+def test_error_during_setup(pytester):
+    pytester.copy_example("examples/test_yield.py")
+    result = pytester.runpytest('-v', '-s')
```

Comment:
Will need to make this explicit if we add a test below

```suggestion
    result = pytester.runpytest('test_yield.py::test_yield', '-v', '-s')
```